mimic/model/nova_objects.py

```diff
@@ -815,6 +816,38 @@ def request_action(self, http_action_request, server_id):
         else:
             return dumps(bad_request("There is no such action currently supported", http_action_request))
 
+    def _create_random_list_of_flavors(self):
+        """
+        Creates a random list of 10 flavors.
+        """
+        for each in range(10):
+            flavor_id = choice([str(each), "onmetal-" + str(each),
+                                "mimic" + str(each)])
+            flavor_name = "{0}GB Instance".format(flavor_id)
+            ram = choice([32768, 131072, 524288, 65536,
```

Comment:
@ClifHouck : ah ok, as the v2 flavors seemed to have the same ram size right. 

I will add this to be something that can be manipulated using the control plane as indicated in #360.